leakcanary-android-instrumentation/src/main/java/leakcanary/Profiler.kt

```diff
@@ -0,0 +1,60 @@
+package leakcanary
```

Comment:
This file is only used by `leakcanary-android-instrumentation/src/androidTest` so it shouldn't be in `leakcanary-android-instrumentation/src/main/`

Reply:
Moved to `/androidTest/`